### DISCUSSION TITLE
magick lossless jpg to avif

### DISCUSSION THREAD

**Discussion by avidseeker** [ImageMagick/ImageMagick#7737] [Help] [answered]
### ImageMagick version

7.1.1-39

### Operating system, version and so on

GNU/Linux

### Description

```sh
$ magick pic.jpg out.avif
```

Is this lossless? If not, is there an option to make it so? 

https://imagemagick.org/script/formats.php#supported doesn't help, and I'm curious to know about imagemagick support for other features of avif (e.g: multi-layered images).

**Comment by snibgo** [comment, marked as answer]
```
magick toes.jpg -quality 100 -define heic:chroma=444 x.avif
```

We can measure the difference:
```
magick compare -metric RMSE toes.jpg x.avif NULL:
1.68261 (2.5675e-05)
```
The images are virtually identical, with an RMSE of 0.0025675%. This very small difference is probably from arithmetical precision of the round-trip sRGB -> YUV -> sRGB.
